src/plugins/swift_hook.py

```diff
@@ -28,7 +39,19 @@ def connection(self):
             options["os_tenant_name"] = connection.host
         yield SwiftService(options=options)
 
-    def list_container(self, container):
+    def list_container(self, container: str) -> Iterator[Dict]:
+        """Returns the items in the objectstore folder (container)
+
+        Args:
+            container (str): The objectstore folder to retreive the files
+
+        Raises:
+            AirflowException: Container cannot be listed
+
+        Yields:
+            Iterator[str]: Iterator[str]: Found file in given objectstor folder (container)
```

Comment:
With type annotations used in the function signature you don't need to repeat them in the docstring. 

Also there is a mismatch between what the function says it returns `Iterator[Dict]` versus what the docstring says it returns `Iterator[str]`